recipes/rtoml/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rtoml" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rtoml-{{ version }}.tar.gz
+  sha256: 6d55fa9df88256c6762f01f18b8bd3f6dc1836600ea8655eb3719a2b8f3df764
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - libgcc-ng  # [linux]
+  host:
+    - pip >=21.0.1
+    - python
+    - setuptools-rust
+  run:
+    - python
+    - libgcc-ng  # [linux]
+
+test:
+  imports:
+    - rtoml
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/samuelcolvin/rtoml
+  summary: A better TOML library for python implemented in rust.
+  license: MIT
+  license_file: LICENSE
```

Comment:
You probably need to include the Rust licenses as well: https://conda-forge.org/docs/maintainer/adding_pkgs.html#licenses-of-included-dependencies